sdcm/nemesis.py

```diff
@@ -583,9 +583,35 @@ def _run_in_cqlsh(self, cmd, node=None):
 
         node.remoter.run('cqlsh {} -e "{}" {}'.format(cql_auth, cmd, node.private_ip_address), verbose=True)
 
-    def _modify_table_property(self, name, val, keyspace="keyspace1", table="standard1"):
+    def select_keyspace_and_table_for_nemesis(self, select_table=False):
+        test_keyspaces = self.cluster.get_test_keyspaces()
```

Comment:
I'm not sure I fully understand the logic here.
If I do, it can be simple randonly select a table like we do in fullscan.